docs/src/tutorials/conic/optimal_power_flow.jl

```diff
@@ -0,0 +1,546 @@
+# ## Initial formulation
+
+# Optimal power flow problems for electrical transmission typically pose the
+# following question: what is the most cost-effective operation of electricity
+# generators while meeting constraints on the safe limits of network components?
+
+# The graph of the network shown here has nine nodes, with three each used for
+# the different purposes of generation ``G`` (nodes 1, 2, and 3), trans-shipment
+# (nodes 4, 6, and 8), and demand ``D`` (nodes 5, 7, and 9).
+
+# ![Nine Nodes](../../assets/case9mod.png)
+
+# We're using the 9-_bus_ network test case `case9mod` to explore this problem.
+# This test case is a version of the [MATPOWER](https://matpower.org/) test case
+# `case9` ([archive](https://github.com/MATPOWER/matpower/tree/master/data))
+# that has been modified by Bukhsh et al. (2013) for their test case archive of
+# optimal power flow problems with local optima. This test case is also
+# extensively evaluated in Krasko and Rebennack (2017).
+
+# Here *bus* and network *node* are taken as analogous terms, as are *branch*
+# and transmission *line*.
+
+# For future reference, let's name the number of nodes in the network:
+
+N = 9;
+
+# The network data can be summarised using a small number of arrays. Using the
+# `sparsevec` function from the `SparseArrays` standard library package, we can
+# give the indices and values of the non-zero data points:
+
+# _Generation power: lower (`lb`) and upper (`ub`)_ bounds
+
+## Active
+P_Gen_lb = SparseArrays.sparsevec([1, 2, 3], [10, 10, 10], N)
+P_Gen_ub = SparseArrays.sparsevec([1, 2, 3], [250, 300, 270], N)
+## Reactive
+Q_Gen_lb = SparseArrays.sparsevec([1, 2, 3], [-5, -5, -5], N)
+Q_Gen_ub = SparseArrays.sparsevec([1, 2, 3], [300, 300, 300], N)
+
+# _Power demand levels (real, reactive, and complex form)_
+P_Demand = SparseArrays.sparsevec([5, 7, 9], [54, 60, 75], N)
+Q_Demand = SparseArrays.sparsevec([5, 7, 9], [18, 21, 30], N)
+S_Demand = P_Demand + im * Q_Demand
+
+# The key decision variables are the real power injections ``P^G`` and reactive
+# power injections ``Q^G``over the allowed range of the generators. All other
+# buses must restrict their generation variables to 0. On the other hand, these
+# non-generator nodes have a fixed  real and reactive power demand, denoted
+# ``P^D`` and ``Q^D`` respectively (these are fixed at 0 in the case of
+# trans-shipment and generator nodes).
+
+# The cost of operating each generator is modeled as a quadratic function of
+# its real power output; in our specific test case, the objective function to
+# minimize is:
+# ```math
+# \begin{align}
+#     \min \;\; & \; 0.11 \;\; (P^G_1)^2 +   5 P^G_1 + 150  \\
+#        + \;\; & 0.085 \; (P^G_2)^2 + 1.2 P^G_2 + 600  \\
+#          \;\; & 0.1225 \;  (P^G_3)^2 +     P^G_3 + 335 \\
+# \end{align}
+# ```
+# Let's create an initial JuMP model with some of this data:
+
+model = Model(Ipopt.Optimizer)
+set_silent(model)
+@variable(model, P_Gen_lb[i] <= P_G[i = 1:N] <= P_Gen_ub[i])
```

Comment:
```suggestion
@variable(model, P_Gen_lb[i] <= P_G[i in 1:N] <= P_Gen_ub[i])
```
I don't know how well I've enforced this in the docs, and JuliaFormatter typically leaves it along because it's in a macro, but the JuMP style guide would prefer ` in ` over ` = `.